src/components/TaskList/TaskList.js

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
 import useStore from '../../services/useStore';
+import Button from '../Button/Button';
 import Task from '../Task/Task';
 
 export default function TaskList() {
 	const tasks = useStore(state => state.tasks);
-
+	const archiveTasks = useStore(state => state.archiveTasks);
+	const archivedTasks = tasks.filter(task => task.archive === false);
```

Comment:
I think the naming of this variable is misleading. The array includes all tasks, that are not(!) archived.

maybe:
- `unArchivedTasks`
- `activeTasks`

Reply:
👍